extensions/discovery-and-network/services.yaml

```diff
@@ -0,0 +1,247 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: services
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: services
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  details: |-
+    header:
+      - name: Controlled by
+        source: metadata.ownerReferences
+        widget: ControlledBy
+      - name: Type
+        source: spec.type
+      - name: Cluster IP
+        source: spec.clusterIP
+      - name: Ports
+        source: >-
+          $count(spec.ports) ? $map(spec.ports, function($value) {$value.port =
+          $value.targetPort ? [$string($value.port), '/', $string($value.protocol)]
+          ~> $join('') : [$string($value.port), ':', $string($value.targetPort),
+          '/', $string($value.protocol)] ~> $join('') }) ~> $join(', ')  : '-'
+      - name: External IPs
+        source: >-
+          $count(status.loadBalancer.ingress) ? $map(status.loadBalancer.ingress,
+          function($value) {$value.ip ? $value.ip : $value.hostname}) ~> $join(',
+          '): ($count($item.spec.externalIPs) ? $item.spec.externalIPs ~> $join(',
+          '): '-')
+    body:
+      - widget: ResourceList
+        source: $subscriptions()
+        name: Subscription
+        children:
+          - source: metadata.name
+            name: Name
+            sort: 'true'
+            widget: ResourceLink
+            resource:
+              name: metadata.name
+              namespace: $root.metadata.namespace
+              kind: kind
+          - source: spec.filter.filters[0].eventType.value
+            name: First event type
+            sort:
+              default: true
+              compareFunction: $compareStrings($first, $second)
+          - source: status.ready
+            name: Ready
+            sort: true
+            widget: Badge
+            highlights:
+              positive:
+                - 'true'
+              negative:
+                - 'false'
+      - widget: ResourceList
+        source: $apirules()
+        name: API Rule
+        sort:
+          - source: spec.service.host
+            default: true
+          - source: spec.service.port
+            compareFunction: $first - $second
+  form: |-
+    - widget: KeyValuePair
+      path: spec.selector
```

Comment:
this field is not displayed anywhere on details view, check how it works and if this works as the rest of selectors provide `podSelector` to details view